app/jsx/components/TabContent1Comp.jsx

```diff
@@ -11,69 +11,108 @@ class TabContent1Comp extends React.Component {
     let p = this.props
     return (
       <div className="c-tabcontent">
-        <ItemActionsComp />
+        <ItemActionsComp status={p.status} content_type={p.content_type} id={p.id} />
```

Comment:
Hmmm.. you still need to define each property individually. Curious how things could get any shorter.